UseLocalCompiler.Directory.Build.props

```diff
@@ -0,0 +1,32 @@
+<Project>
+  <PropertyGroup>
+    <LocalFSharpCompilerConfiguration>Debug</LocalFSharpCompilerConfiguration>
+
+    <LocalFSharpCompilerPath>$(MSBuildThisFileDirectory)</LocalFSharpCompilerPath>
+    <LocalFSCDllPath>$(LocalFSharpCompilerPath)/artifacts/bin/fsc/$(LocalFSharpCompilerConfiguration)/net8.0/fsc.dll</LocalFSCDllPath>
+
+    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+
+    <DisableAutoSetFscCompilerPath>true</DisableAutoSetFscCompilerPath>
+
+    <DotnetFscCompilerPath>$(LocalFSCDllPath)</DotnetFscCompilerPath>
+    <Fsc_DotNET_DotnetFscCompilerPath>$(LocalFSCDllPath)</Fsc_DotNET_DotnetFscCompilerPath>
+
+    <FSharpPreferNetFrameworkTools>False</FSharpPreferNetFrameworkTools>
+    <FSharpPrefer64BitTools>True</FSharpPrefer64BitTools>
+
+    <LocalFSharpBuildBinPath>$(LocalFSharpCompilerPath)/artifacts/bin/fsc/$(Configuration)/net8.0</LocalFSharpBuildBinPath>
+    <FSharpBuildAssemblyFile>$(LocalFSharpBuildBinPath)/FSharp.Build.dll</FSharpBuildAssemblyFile>
+    <FSharpTargetsPath>$(LocalFSharpBuildBinPath)/Microsoft.FSharp.Targets</FSharpTargetsPath>
+    <FSharpPropsShim>$(LocalFSharpBuildBinPath)/Microsoft.FSharp.NetSdk.props</FSharpPropsShim>
+    <FSharpTargetsShim>$(LocalFSharpBuildBinPath)/Microsoft.FSharp.NetSdk.targets</FSharpTargetsShim>
+    <FSharpOverridesTargetsShim>$(LocalFSharpBuildBinPath)/Microsoft.FSharp.Overrides.NetSdk.targets</FSharpOverridesTargetsShim>
+  </PropertyGroup>
+
+  <UsingTask TaskName="FSharpEmbedResourceText" AssemblyFile="$(FSharpBuildAssemblyFile)" Override="true" />
+  <UsingTask TaskName="FSharpEmbedResXSource" AssemblyFile="$(FSharpBuildAssemblyFile)" Override="true" />
```

Comment:
This has to go into .targets , I am afraid it will fail or not do anything if inside .props

Reply:
It seems to be working